lib/jobs/ssh-job.js

```diff
@@ -0,0 +1,207 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+module.exports = sshJobFactory;
+
+//var Client = require('ssh2').Client;
+
+di.annotate(sshJobFactory, new di.Provide('Job.Ssh'));
+di.annotate(sshJobFactory, new di.Inject(
+    'Job.Base',
+    'JobUtils.CommandParser',
+    'Util',
+    'Logger',
+    'Assert',
+    'Promise',
+    'Services.Waterline',
+    'Services.Encryption',
+    'ssh',
+    '_'
+));
+
+function sshJobFactory(
+    BaseJob,
+    parser,
+    util,
+    Logger,
+    assert,
+    Promise,
+    waterline,
+    cryptService,
+    ssh,
+    _
+) {
+    var logger = Logger.initialize(sshJobFactory);
+    var Client = ssh.Client;
+    function SshJob(options, context, taskId) {
+        SshJob.super_.call(this, logger, options, context, taskId);
+        assert.string(this.context.target);
+        this.nodeId = this.context.target;
+        this.acceptedCodes = (options.acceptedCodes || []).concat(0);
```

Comment:
Examine the task [catalog-bmc](https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/catalog-bmc.js) and [catalog-mgmt-bmc](https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/catalog-mgmt-bmc.js), the `acceptedCodes` is assigned for each command, that will be more flexible though a little complex as not every command in the list will always share the same `accpetedCodes`. So I suggest copying catalog-bmc design.

Meanwhile, to keep the options name consistent across jobs, I also suggest renaming the option from `acceptedCodes` to `acceptedResponseCodes`.